src/huggingface_hub/hf_api.py

```diff
@@ -1309,8 +1333,10 @@ def update_repo_visibility(
         r.raise_for_status()
         return r.json()
 
+    @_deprecate_positional_args
     def move_repo(
         self,
+        *,
         from_id: str,
         to_id: str,
```

Comment:
I'm on the fence with this one, what do you think @osanseviero?

Reply:
For consistency with others where `repo_id` is before `*`, I would do `from_id` and `to_id` first. 

No strong opinion though, I could easily see users getting them the other way around so being more explicit can help.